src/components/Match/MatchStory.jsx

```diff
@@ -550,10 +561,24 @@ class TimeMarkerEvent extends StoryEvent {
   }
   format() {
     return [
-      <h3 key={`minute_${this.minutes}_subheading`}>
+      <h3 key={`minute_${this.minutes}_subheading`} style={{ marginBottom: 0 }}>
         {formatTemplate(strings.story_time_marker, { minutes: this.minutes })}
       </h3>,
-      <hr key={`minute_${this.minutes}_hr`} />,
+      <div key={`minute_${this.minutes}_networth_text`} className={styles.storyNetWorthText}>
+        <div style={{ width: `${this.radiant_percent}%` }}>
+          {GoldSpan(this.radiant_gold)}
+        </div>
+        <div style={{ color: this.radiant_gold > this.dire_gold ? styles.green : styles.red, left: `${this.radiant_percent}%` }}>
+          {Math.abs(this.radiant_percent - this.dire_percent)}% / {GoldSpan(Math.abs(this.radiant_gold - this.dire_gold))} Diff
```

Comment:
Whoops.
Also, should this be "adv" or some other shortened version of "advantage" since we are specifying who has the advantage via color?